modules/managing-memcached-operator-using-olm.adoc

```diff
@@ -64,6 +64,10 @@ $ kubectl apply -f deploy/rbac.yaml
 $ kubectl apply -f deploy/crd.yaml
 ----
 +
+NOTE: The `rbac.yaml` and `crd.yaml` files were generated into the `deploy/`
+directory by the Operator SDK in the previous section when the Memcached
```

Comment:
I'd consider removing "in the previous section."

Reply:
Yeah I wasn't sure about that, but I wasn't sure how else to make it clear that they needed to complete that other section (without linking :) ). And also earlier in this file it had already made a reference to "the previous section". wdyt?

Reply:
I think that the prereq to have built the memcached operator (line 23) covers it. Assemblies are meant to be 1:1 mappings to user stories, so I think it's fair to assume that the user will have completed the previous sections, especially when it's reinforced by the prereqs. (I'm not a fan of the other "previous section" reference, either, for what it's worth.)

Reply:
Yeah, it's just that this whole BZ was because it was felt like someone might come right into this section without really completing the above section. But I'll go ahead and remove the "previous section" reference. Thanks.

Reply:
@kalexand-rh I removed it and also reworded to "you built" so that hopefully makes it more clear that the user actually had to do something else. Let me know if it looks okay now.

Reply:
That sounds good to me. Thank you!